marklogic-data-hub/src/main/resources/ml-modules/root/com.marklogic.hub/lib/hub-entities.xqy

```diff
@@ -187,84 +75,39 @@ declare function hent:wrap-duplicates(
-      let $invalid-values := "reject"
-      let $_ := map:put($ri-map, "invalid-values", $invalid-values)
-      let $_ := map:put($ri-map, "path-expression", "/*:envelope/*:instance/" || $entity-type-name || "/" || $range-index-property)
-      let $_ := map:put($ri-map, "range-value-positions", false())
-      let $_ := map:put($ri-map, "scalar-type", $datatype)
-      return json:array-push($range-path-indexes, $ri-map)
-      ,
-      let $word-lexicon-properties := map:get($entity-type, "wordLexicon")
-      for $word-lexicon-property in $word-lexicon-properties ! json:array-values(.)
-      let $wl-map := json:object()
-      let $property := map:get($properties, $word-lexicon-property)
-      let $collation := head( (map:get($property, "collation"), "http://marklogic.com/collation/en") )
-      let $_ := map:put($wl-map, "collation", $collation)
-      let $_ := map:put($wl-map, "localname", $word-lexicon-property)
-      let $_ := map:put($wl-map, "namespace-uri", "")
-      return json:array-push($word-lexicons, $wl-map)
-      ,
-      let $primary-key-property := map:get($entity-type, "primaryKey")
-      where $primary-key-property
-      return
-        let $pk-map := json:object()
-        let $property := map:get($properties, $primary-key-property)
-        let $specified-datatype := es-wrapper:resolve-datatype($model, $entity-type-name, $primary-key-property)
-        let $datatype := es-wrapper:indexable-datatype($specified-datatype)
-        let $collation := head( (map:get($property, "collation"), "http://marklogic.com/collation/en") )
-        let $_ := map:put($pk-map, "collation", $collation)
-        let $_ := map:put($pk-map, "localname", $primary-key-property)
-        let $_ := map:put($pk-map, "namespace-uri", "")
-        let $_ := map:put($pk-map, "range-value-positions", fn:false())
-        let $_ := map:put($pk-map, "scalar-type", $datatype)
-        let $_ := map:put($pk-map, "invalid-values", "reject")
-        return json:array-push($range-element-indexes, $pk-map)
-    )
-  let $path-namespaces := json:array()
-  let $pn := json:object()
-  let $_ := map:put($pn, "prefix", "es")
-  let $_ := map:put($pn, "namespace-uri", "http://marklogic.com/entity-services")
-  let $_ := json:array-push($path-namespaces, $pn)
-  let $database-properties := json:object()
-  let $_ := map:put($database-properties, "path-namespace", $path-namespaces)
-  let $_ := map:put($database-properties, "element-word-lexicon", $word-lexicons)
-  let $_ := map:put($database-properties, "range-path-index", $range-path-indexes)
-  let $_ := map:put($database-properties, "range-element-index", $range-element-indexes)
-  return xdmp:to-json($database-properties)
+    for $x in ("database-name", "schema-database", "triple-index", "collection-lexicon")
+    return
+      map:delete($o, $x)
+  let $_ :=
+    for $idx in json:array-values(map:get($o, "range-path-index"))
+    let $_ := xdmp:log(("idx:", $idx))
```

Comment:
is it still useful to have this xdmp:log statement?

Reply:
no. it should go away